app/src/main/java/ch/sdp/vibester/activity/DeleteSongsActivity.kt

```diff
@@ -125,6 +126,38 @@ class DeleteSongsActivity : AppCompatActivity() {
         return false
     }
 
+    private fun removeFromProperties(buttonText: String): Boolean {
+        var properties = File(applicationContext.getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS), "properties.txt")
+        var tempProp = File(applicationContext.getExternalFilesDir(Environment.DIRECTORY_DOWNLOADS), "tempProp.txt")
+
+        var propReader = BufferedReader(FileReader(properties))
+        var propWriter = BufferedWriter(FileWriter(tempProp, true))
+
+        var currentLine = propReader.readLine()
+        val buttonSplit = buttonText.trim().split("-")
+
+        while (currentLine != null) {
+            var trimmed = currentLine.trim()
+            val split = trimmed.split(" - ")
+
+            if (split[0].trim().lowercase() == buttonSplit[0].trim().lowercase()
```

Comment:
Maybe add some comments explaining what split[0] and split[1] are?

Reply:
Done, can you check if the explanations are clear?